src/api.ts

```diff
@@ -304,22 +304,23 @@ router.post(
       // Once confirmed, move to archive collection
       await database.cancelBid(bidData, archiveCollection);
 
-      // const message: TypedMessage<BidCancelledV1Data> = {
-      //   event: MessageType.BidCancelled,
-      //   version: "1.0",
-      //   timestamp: new Date().getTime(),
-      //   logIndex: undefined,
-      //   blockNumber: undefined,
-      //   data: {
-      //     account: signer,
-      //     uniqueBidId: bidData.uniqueBidId,
-      //   },
-      // };
+      const message: TypedMessage<BidCancelledV1Data> = {
+        event: MessageType.BidCancelled,
+        version: "1.0",
+        timestamp: new Date().getTime(),
+        logIndex: undefined,
+        blockNumber: undefined,
+        data: {
+          account: signer,
+          uniqueBidId: bidData.uniqueBidId,
+        },
+      };
 
-      // await queue.sendMessage(message);
+      await queue.sendMessage(message);
+      // await queue.close(); 
 
       return res.status(200).send();
-    } catch (e) {
+    } catch (e: any) { // have to type message?
```

Comment:
Sadly I don't think you can't type this in the catch. If you need error specific logic you can use type guards to narrow.
More info: https://fettblog.eu/typescript-typing-catch-clauses/#2.-there-is-only-one-catch-clause-in-javascript

Reply:
My TS errors this saying I have to strangely